cicd/jenkins/Jenkinsfile

```diff
@@ -852,7 +878,7 @@ PolicyDocument
             steps {
               script {
                 sh "rm -rf dist"
-                unstash name: "DIST-CHI"
+                unstash name: "DIST-VUE-DOCUMENTATION"
```

Comment:
Changing temporarily for testing purposes. Will bring the standard Chi compilation back later.